app/renderer/js/tray.ts

```diff
@@ -21,6 +21,8 @@ const iconPath = (): string => {
 	return APP_ICON + (process.platform === 'win32' ? 'win.ico' : 'macOSTemplate.png');
 };
 
+const winTrayIconPath = (): string => APP_ICON + 'win.png';
```

Comment:
This filename and variable name make no sense for an icon that’s specifically for the unread case. Also, it should be in `.ico` format for Windows.

Reply:
Oh Cool , Yeah it should have been in a <code>.ico</code> format , Will make necessary changes 👍

Reply:
@andersk , Am I missing out something?